<launch>

 <master auto="start"/>     
   
    <param name ="/use_sim_time" value="false"/>
    <arg name ="target_frame" default="trunk"/>
    <arg name ="camera1" default="/camera1/point_cloud_face" />
    <arg name ="camera3" default="/camera3/point_cloud_left" />
    <arg name ="camera4" default="/camera4/point_cloud_right" />
  <!-- point cloud to laser -->
        <!-- <include file="$(find pointcloud_to_laserscan)/launch/go1_pc_to_laser.launch"></include> -->
 <!-- launch the waypoints nav -->
        <include file="$(find fulanghua_waypoints_nav)/launch/nav.launch"></include>

 <!--- Run Waypoints -->
        <include file="$(find orne_waypoints_editor)/launch/edit_waypoints_viz.launch"></include>
 
 <node pkg="unitree_go1" type="posture_srv" name="go1_posture" output="screen">
	<param name="cmd_vel" value="$(arg cmd_vel)"/>
  </node>
   <node pkg="unitree_go1" type="pcl_node" name="pcl_node1" output="screen">
       <param name="target_frame" value="$(arg target_frame)"/>
       <param name="topic_name" value="$(arg camera1)"/>
   </node>
   <node pkg="unitree_go1" type="pcl_node" name="pcl_node3" output="screen">
       <param name="target_frame" value="$(arg target_frame)"/>
       <param name="topic_name" value="$(arg camera3)"/>
   </node>
   <node pkg="unitree_go1" type="pcl_node" name="pcl_node4" output="screen">
       <param name="target_frame" value="$(arg target_frame)"/>
       <param name="topic_name" value="$(arg camera4)"/>
   </node>


  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find unitree_go1)/rviz_cfg/unitree_rviz.rviz"></node>

</launch>
